apps/web/src/services/ops-flow/pages/TaskDetailPage.vue

```diff
@@ -190,10 +193,48 @@ const { isSuccess, mutateAsync: updateTaskMutation, isPending: isUpdating } = us
     },
 });
 
+/* update task description */
+const { isSuccess: isSuccessUpdateDescription, mutateAsync: updateDescriptionMutation, isPending: isUpdatingDescription } = useMutation<TaskModel, APIError>({
+    mutationFn: async () => {
+        if (!task.value) throw new Error('Origin task is not defined');
+        const res = await taskAPI.updateDescription({
+            task_id: task.value.task_id,
+            description: taskContentFormStore.getters.defaultData[DEFAULT_FIELD_ID_MAP.description],
+        });
+        return res;
+    },
+    onSuccess: () => {
+        queryClient.invalidateQueries({ queryKey: taskDetailQueryKey.value });
+        showSuccessMessage(_i18n.t('OPSFLOW.ALT_S_UPDATE_TARGET', { target: taskManagementTemplateStore.templates.task }), '');
+        refetchEvents();
+    },
+    onError: (error) => {
+        ErrorHandler.handleRequestError(error, _i18n.t('OPSFLOW.ALT_E_UPDATE_TARGET', { target: taskManagementTemplateStore.templates.task }));
+    },
+});
+
+
+
+
 /* form button handling */
 const handleSaveChanges = async () => {
     if (!taskContentFormStore.getters.isAllValid) return;
-    await updateTaskMutation();
+
+    const currentDescription = taskContentFormStore.getters.defaultData[DEFAULT_FIELD_ID_MAP.description];
+    const isDescriptionChanged = task.value?.description !== currentDescription;
+    if (isDescriptionChanged) {
+        await updateDescriptionMutation();
+    }
+
+    const isDataChanged = !isEqual(task.value?.data, taskContentFormStore.getters.data);
+    const otherFields = Object.entries(taskContentFormStore.getters.defaultData).filter(([k]) => k !== DEFAULT_FIELD_ID_MAP.description);
+    const isOtherFieldsChanged = otherFields.some(([k, v]) => {
+        if (k === DEFAULT_FIELD_ID_MAP.description) return false;
+        return task.value?.[k] !== v;
+    });
+    if (isDataChanged || isOtherFieldsChanged) {
+        await updateTaskMutation();
+    }
```

Comment:
suggestion: I think it would be better to use hasUnsavedChanges state in the store.